CHANGELOG.md

```diff
@@ -1,5 +1,77 @@
+* bugtool: Dump envoy metrics for troubleshooting (Backport PR #23779, Upstream PR #22797, @sayboras)
+* chore(deps): update actions/checkout action to v3.3.0 (v1.12) (#23994, @renovate[bot])
+* chore(deps): update all github action dependencies (v1.12) (patch) (#23993, @renovate[bot])
+* chore(deps): update dependency cilium/hubble to v0.11.2 (v1.12) (#23909, @renovate[bot])
+* chore(deps): update docker.io/library/ubuntu:20.04 docker digest to 4a45212 (v1.12) (#23693, @renovate[bot])
+* chore(deps): update docker.io/library/ubuntu:20.04 docker digest to 9fa30fc (v1.12) (#24137, @renovate[bot])
+* chore(deps): update quay.io/cilium/hubble docker tag to v0.11.2 (v1.12) (#23923, @renovate[bot])
+* clustermesh, kvstore: consistently pass controller context to kvstore operations (Backport PR #23779, Upstream PR #23333, @tklauser)
+* docs: correct Prometheus port (Backport PR #23779, Upstream PR #23404, @lizrice)
+* docs: Document CONFIG_PERF_EVENTS requirement (Backport PR #24197, Upstream PR #24055, @joestringer)
+* docs: Drop sphinxcontrib-openapi fork, switch back to upstream (Backport PR #23779, Upstream PR #23118, @qmonnet)
+* docs: Fix the dead link to Mellanox performance tuning guide (Backport PR #24083, Upstream PR #24012, @gentoo-root)
+* docs: Mark Git repository as safe, at runtime, if in a container (Backport PR #24067, Upstream PR #21069, @qmonnet)
+* docs: replace usage of api.twitter.com (Backport PR #23779, Upstream PR #23669, @kaworu)
+* Enable Google Analytics 4 (Backport PR #24067, Upstream PR #22220, @chalin)
+* fix(deps): update module golang.org/x/net to v0.7.0 [security] (master) (Backport PR #23957, Upstream PR #23904, @renovate[bot])
+* Fixed link to broken anchor in RKE doc (Backport PR #23779, Upstream PR #23706, @raphink)
+* Introduce node IDs in the datapath and the agent, so datapath can later use them to identify remote nodes (Backport PR #23779, Upstream PR #23202, @pchaigno)
+* IPsec: Remove `IP_POOLS` logic (Backport PR #24083, Upstream PR #24030, @pchaigno)
+* Node ID restoration (Backport PR #23779, Upstream PR #23578, @pchaigno)
+* Remove / in RKE doc link as it causes redirect bug (Backport PR #23779, Upstream PR #23728, @raphink)
+* workflow: fixes LLVM, Clang cache and install path (Backport PR #23779, Upstream PR #23740, @brlbil)
+
+**Other Changes:**
+* agent: dump stack on stale probes [backport-1.12] (#24213, @squeed)
+* docs: Add note for operator.extraEnv (#23843, @sayboras)
+* install: Update image digests for v1.12.7 (#23738, @joestringer)
+* Revert "Pick up etcd v3.5.7" (#23788, @michi-covalent)
+* update images 1.12 (#24303, @nebril)
+* v1.12 - Backport initContainer change (#24332, @ferozsalam)
+* v1.12 backport: fix cgroup program detachment and 1.14 downgrade (#24183, @ti-mo)
+* v1.12 Backports 2023-03-03 (#24155, @jibi)
+* v1.12 Backports 2023-03-14 (#24369, @nebril)
+* v1.12 Backports 2023-03-15 (#24386, @nebril)
```

Comment:
We should replace with their own PR title, not sure why the tooling didn't do that

Reply:
Those three backport PRs have a second block after the list of PRs so I'm guessing that broke the release script?
![image](https://user-images.githubusercontent.com/1764210/225441842-caddf48f-2575-49ed-a734-3ce8d39141ae.png)

Introduced by @aanm in https://github.com/isovalent/backporter/commit/3c01890e73e063dba8f14708f8faa0aa3c0c4a5e.

Reply:
updated CHANGELOG.md and the PR description.